fab/inventory/tsung

```diff
@@ -0,0 +1,36 @@
+[proxy]
+hqproxy4.internal.commcarehq.org
+
+[webworkers]
+hqdjango4-staging.internal.commcarehq.org
+hqdjango5-staging.internal.commcarehq.org
+
+[couchdb]
+hqdb0-staging.internal.commcarehq.org
+
+[postgresql]
+hqdb0-staging.internal.commcarehq.org
+
+[rabbitmq]
+hqdb0-staging.internal.commcarehq.org
+
+[zookeeper]
+hqkafka0-staging.internal.commcarehq.org
+
+[kafka]
+hqkafka0-staging.internal.commcarehq.org kafka_broker_id=843
```

Comment:
similar to below, i think we do need to publish to kafka on submission but we could potentially stick it on another machine since it will be doing very little work.